src/app/containers/Include/index.test.jsx

```diff
@@ -77,4 +93,63 @@ describe('IncludeContainer', () => {
     );
     expect(container).toMatchSnapshot();
   });
+
+  const runningIncludeTest = includeType => {
+    it(`should add require to the page for ${includeType}`, async () => {
+      render(
+        <IncludeContainerWithMockContext
+          toggleState={toggleStateFalse}
+          html={fakeMarkup}
+          type={includeType}
+        />,
+      );
+
+      await waitFor(() =>
+        expect(
+          Array.from(document.querySelectorAll('head script')),
```

Comment:
Could this potentially break if more scripts are potentially added to the head in the future outside of includes?

Reply:
I think that's a fair point, we should make the assertions more precise, looking for specifically the requirejs script and the inline `require.config`

Reply:
@HarveyPeachey , I've made the assertions more specific, they still check the length of the head scripts array, but hopefully it reads more clearly what is being asserted for: https://github.com/bbc/simorgh/pull/6347/files/65b10d92e21d85bb90682ff89cd094882371d5c9..233bba65cca06401d65b296a9e7e076630db1b0a